articles/install-guide/vs-code.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Installing the Q# development environment for VS Code
+author: chgranad
+uid: microsoft.quantum.install.vs-code
+ms.author: christopher.granade@microsoft.com
+ms.date: 7/31/2018
+ms.topic: article
+---
+
+# Installing the Quantum Development Kit for Visual Studio Code #
+
+The Quantum Development Kit uses the .NET Core SDK (2.0 or later) to make it easy to create, build, and run Q# projects from the command line.
+We recommend using the Quantum Development Kit together with Visual Studio Code.
+
+1. Please see [Getting started with .NET](https://www.microsoft.com/net/learn/get-started) for how to download and install the .NET Core SDK.
```

Comment:
As you point out, some people gets confused when we send them to "Getting started with .NET". Why don't we point them instead to:
https://www.microsoft.com/net/download
and ask them to download the **.NET Core SDK**

Reply:
Sounds good, I'll change that accordingly, then.